oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java

```diff
@@ -1226,7 +1260,10 @@ private boolean isDeletedOrOrphanedNode(final NodeState traversedState, final Pa
                 phases.stop(GCPhase.FULL_GC_COLLECT_ORPHAN_NODES);
                 return false;
             }
+
+            // first check if gapOrphansModeEnabled is set in fullGCOptions, then check in fullGCMode enum
```

Comment:
not sure I see the purpose of this comment, would suggest to remove

Reply:
fixed, thank you for the comment

Reply:
but the comment is still here?